app/assets/stylesheets/pages/_machines.scss

```diff
@@ -0,0 +1,270 @@
+  }
+
+  .device-selected {
+    background-color: $c-pink;
+    box-shadow: 3px 4px 19px rgba(0, 0, 0, 0.25);
+    position: relative;
+
+    a {
+      color: white;
+    }
+
+    &:after,
+    &:before {
+      border: solid transparent;
+      content: " ";
+      height: 0;
+      left: 50%;
+      pointer-events: none;
+      position: absolute;
+      top: 100%;
+      width: 0;
+    }
+
+    &:after {
+      border-top-color: $c-pink;
+      border-width: 15px;
+      margin-left: -15px;
+    }
+
+    &:before {
+      border-top-color: $c-pink;
+      border-width: 10px;
+      margin-left: -10px;
+    }
+  }
+}
+
+.device__problems {
+  @include clearfix();
+  padding: 1rem 0;
+
+  article {
+    border-bottom: 1px solid lighten($c-text, 50%);
+  }
+
+  header {
+    @include clearfix();
+    margin: 0 auto;
+    max-width: 600px;
+    padding: 1rem 0;
+    width: rd(355, 415);
+
+    &:hover,
+    &:focus {
+      h3 {
+        color: $c-pink;
+      }
+    }
+  }
+
+  header,
+  img {
+    cursor: pointer;
+  }
+
+  .__inactive-list-item {
+    border: 0;
+    clip: rect(0 0 0 0);
+    display: none;
+    height: 1px;
+    margin: -1px;
+    overflow: hidden;
+    padding: 0;
+    position: absolute;
+    width: 1px;
+  }
+
+  h3 {
+    @include fluid-size("font-size", 20px, 24px);
+    float: left;
+    font-family: $f-body;
+    font-weight: 400;
+    transform-origin: left;
+    transition: all 0.3s ease-in-out;
+  }
+
+  button {
+    background: none;
+    border: none;
+    float: right;
+    transition: all 0.3s ease;
+  }
+
+  .-flip-caret {
+    transform: rotate(180deg);
+  }
+
+  @include breakpoint(min-width 950px) {
+    ul {
+      @include clearfix();
+      margin: 0 auto;
+      max-width: 700px;
+      width: rd(682, 1024);
+    }
+
+    header {
+      max-width: 100%;
+      padding: 0 0 rem(24) 0;
+      width: 100%;
+    }
+
+    p {
+      margin: 0;
+      width: 100%;
+    }
+
+    li {
+      margin: 0 0 rem(48) 0;
+      width: 95%;
+    }
+
+    article {
+      margin: 0 auto;
+    }
+  }
+
+  @include breakpoint(min-width 1024px) {
+    padding: rem(85) 0;
+  }
+}
+
+.problem__solutions {
+  margin: 0 auto;
+  position: relative;
+
+  img,
+  svg {
+    display: block;
+    height: auto;
+    margin: 0 auto 1.5rem auto;
+    min-width: 52px;
+    width: rd(340, 415);
+  }
+
+  p {
+    @include fluid-size("font-size", 16px, 18px);
+    line-height: 1.6;
+    margin: 0 auto;
+    width: 85%;
+  }
+}
+
+.__inactive {
+  display: none;
+  height: 0;
+  opacity: 0;
+  width: 0;
+}
+
+@keyframes show {
+  from {
+    opacity: 0;
+    transform: translate(0px, -100px);
+  }
+
+  to {
+    height: 100%;
+    opacity: 1;
+    transform: translate(0px, 0px);
+  }
+}
+
+.shown {
+  animation: show 0.5s both ease-in-out;
+  padding: 0 0 5vh 0;
+}
+
+@include breakpoint(min-width 1200px) {
+  .two-column {
+    max-width: 1500px;
+
+    ul {
+      width: 45%;
+    }
+  }
+
+  .problem-columns {
```

Comment:
your `ul` and `.problem-columns` are the same element - try to make that more clear. Like:

```
ul {
    .two-column & {
        // styles for .two-column ul
    }

    &.problem-columns {
        // column styles
    }
}
```